go/vt/schemadiff/table.go

```diff
@@ -1284,9 +1297,69 @@ func (c *CreateTableEntity) diffColumns(alterTable *sqlparser.AlterTable,
 				}
 			}
 			expectAppendIndex++
-			alterTable.AlterOptions = append(alterTable.AlterOptions, addColumn)
+			addColumns = append(addColumns, addColumn)
+		}
+	}
+	renameColumns := []*sqlparser.RenameColumn{}
+	if hints.ColumnRenameStrategy == ColumnRenameHeuristicStatement {
```

Comment:
Would it be nicer to extract the below logic into a separate method for easier reading etc?